apps/style/common.scss

```diff
@@ -289,6 +289,7 @@ button.arrow:disabled {
   font-size: 14px;
   overflow: hidden;
   min-height: calc(50% - #{($workspace-headers-height / 2) + 1}) !important;
+  direction: ltr;
```

Comment:
codeTextbox will only ever refer to droplet code, right (not blockly)? assuming this is true, lgtm